app/models/application_shelter.rb

```diff
@@ -0,0 +1,20 @@
+class ApplicationShelter < ApplicationRecord
```

Comment:
Let's look through our db and see if we can accomplish the same things without creating a whole new model/migration.  I feel like there's a way to access all the data we need through our other associations, but am not 100%.